src/org/parosproxy/paros/network/ConnectionParam.java

```diff
@@ -223,6 +225,13 @@ protected void parse() {
         }
         
         try {
+            this.httpStateEnabled = getConfig().getBoolean(HTTP_STATE_ENABLED, true);
+        } catch (ConversionException e) {
+            log.error("Error while loading the option httpStateEnabled: " + e.getMessage(), e);
+        }
+        
+        

```

Comment:
Seems to be something weird with the indentation here, I'm guessing there's mixed spaces and tabs.

Also the extra blank line should be removed.